src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -30,14 +29,36 @@ public static class MemoryInformation
         private static readonly byte[] Committed_AS = Encoding.UTF8.GetBytes("Committed_AS:");
 
         private static readonly int ProcessId;
+        private static readonly bool WindowsSupportsMemoryCountersEx2;
+        private static readonly Size? WindowsInstalledMemory;
 
         public static readonly Size TotalPhysicalMemory;
 
         static MemoryInformation()
         {
             using (var process = Process.GetCurrentProcess())
+            {
                 ProcessId = process.Id;
 
+                if (PlatformDetails.RunningOnWindows)
+                {
+                    // supported only on Windows 10 22H2 with September 2023 cumulative update or Windows 11 22H2 with September 2023 cumulative update and above
```

Comment:
what is the Windows Server support for this?

Reply:
Supported in Windows Server 2025.